lib/spoom/rbs.rb

```diff
@@ -4,22 +4,64 @@
 module Spoom
   module RBS
     class Comments
-      #: Array[Annotations]
+      #: Array[Annotation]
       attr_reader :annotations
 
       #: Array[Signature]
       attr_reader :signatures
 
       #: -> void
       def initialize
-        @annotations = [] #: Array[Annotations]
+        @annotations = [] #: Array[Annotation]
         @signatures = [] #: Array[Signature]
       end
 
       #: -> bool
       def empty?
         @annotations.empty? && @signatures.empty?
       end
+
+      #: -> Array[Annotation]
+      def class_annotations
+        @annotations.select do |annotation|
+          case annotation.string
+          when "@abstract"
+            true
+          when "@interface"
+            true
+          when "@sealed"
+            true
+          when "@final"
+            true
+          when /^@requires_ancestor: /
+            true
```

Comment:
I think we can put multiple cases together like:

```suggestion
          when "@abstract", "@interface", "@sealed", "@final"
            true
          when /^@requires_ancestor: /
            true
```

Regexp and string matches can be put together too, but I think giving the regexp match its own condition can make it easier to spot.